### PR TITLE
Variety of copy & URL/I changes

### PR DESCRIPTION
**Pull request by siemvaessen**
Variety of copy & URL/I changes, nothing major.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -100,11 +100,11 @@ workflows:
       #           name: Set env variables
       #           command: |
       #             if [ "${CIRCLE_BRANCH}" == "master" ]; then
-      #               echo 'export CYPRESS_BASE_URL="https://iatidatastore.iatistandard.org"' >> $BASH_ENV
+      #               echo 'export CYPRESS_BASE_URL="https://iati.cloud"' >> $BASH_ENV
       #             elif [ "${CIRCLE_BRANCH}" == "develop" ]; then
-      #               echo 'export CYPRESS_BASE_URL="https://store.staging.iati.cloud"' >> $BASH_ENV
+      #               echo 'export CYPRESS_BASE_URL="https://iati.cloud""' >> $BASH_ENV
```
Comment (sylvanr) [inline .circleci/config.yml:105]: extra " remaining here.

File: .circleci/config.yml [hunk 2]
```diff
@@ -100,11 +100,11 @@ workflows:
...
       #             else
-      #               echo 'export CYPRESS_BASE_URL="https://store.staging.iati.cloud"' >> $BASH_ENV
+      #               echo 'export CYPRESS_BASE_URL="https://iati.cloud""' >> $BASH_ENV
```
Comment (sylvanr) [inline .circleci/config.yml:107]: extra " remaining here.

File: README.md [hunk 3]
```diff
@@ -16,13 +16,13 @@ IATI is a global aid transparency standard and it makes information about aid sp
 ## About the project
 * Website:         <a href="https://www.iati.cloud" target="_blank">www.iati.cloud</a>
 * Authors:          <a href="https://www.zimmerman.team/" target="_blank">Zimmerman</a>
-* License:          AGPLv3 (see included <a href="https://github.com/zimmerman-zimmerman/query-builder-iati.cloud/blob/develop/LICENSE.MD" target="_blank">LICENSE</a> file for full license)
-* Github Repo:      <a href="https://github.com/zimmerman-zimmerman/query-builder-iati.cloud/" target="_blank">github.com/zimmerman-zimmerman/query-builder-iati.cloud</a>
-* Bug Tracker:      <a href="https://github.com/zimmerman-zimmerman/query-builder-iati.cloud/issues" target="_blank">github.com/zimmerman-zimmerman/query-builder-iati.cloud/issues</a>
+* License:          AGPLv3 (see included <a href="https://github.com/zimmerman-team/query-builder-iati.cloud/blob/develop/LICENSE.MD" target="_blank">LICENSE</a> file for full license)
+* Github Repo:      <a href="https://github.com/zimmerman-team/query-builder-iati.cloud/" target="_blank">github.com/zimmerman-zimmerman/query-builder-iati.cloud</a>
```
Comment (sylvanr) [inline README.md:20]: isn't this url supposed to be
zimmerman-team/iati.cloud.frontend ?

File: public/index.html [hunk 4]
```diff
@@ -21,37 +21,12 @@
...
-      u.parentNode.insertBefore(s, u);
-    })(window, document, 'script', '//cdn.sematext.com/rum.js', 'strum');
-  </script>
-  <script type="text/javascript">
-    strum('config', {
-      token: '155abb23-9f7e-4bf0-bebd-86ee4043ed74',
-      receiverUrl: 'https://rum-receiver.eu.sematext.com',
-    });
-  </script>
+
```
Comment (sylvanr) [inline public/index.html:28]: guessing these enters are left over accidentally, lets remove them to keep the file clean

File: src/app/modules/landing-module/common/LandingMidSection.tsx [hunk 5]
```diff
@@ -11,8 +11,8 @@ export const LandingMidSection = () => {
           <Grid item sm={12} md={6} lg={4}>
             <ArticleCard
               loaded
-              title="About Datastore"
-              description="The Datastore makes use of iati.cloud, an open-source native IATI data service developed by Zimmerman. IATI.cloud extracts and stores raw IATI XML files from the IATI Registry and makes it available as API endpoints to build data driven information solutions."
+              title="About IATI.cloud"
+              description="IATI.cloud, an open-source native IATI data service developed by Zimmerman. IATI.cloud extracts and stores raw IATI XML files from the IATI Registry and makes it available as API endpoints to build data driven information solutions."
```
Comment (sylvanr) [inline src/app/modules/landing-module/common/LandingMidSection.tsx:15]: sentence no longer makes sense. Maybe replace the comma with 'is'

**Review by sylvanr — CHANGES_REQUESTED** [review]
just some changes for clarity

File: README.md (hunk 3 above)
Comment (siemvaessen) [reply README.md:20]: yes!

File: public/index.html (hunk 4 above)
Comment (siemvaessen) [reply public/index.html:28]: gracias

**Review by juanalb — APPROVED** [review]

**Review by sylvanr — APPROVED** [review]

**Pull request merged**